dhis-2/dhis-services/dhis-service-analytics/src/test/java/org/hisp/dhis/analytics/common/CommonQueryRequestMapperTest.java

```diff
@@ -127,7 +128,7 @@ void mapWithSuccessOnlyDimension()
         when( programService.getPrograms( aCommonQueryRequest.getProgram() ) ).thenReturn( programs );
         when( i18nManager.getI18nFormat() ).thenReturn( anyI18nFormat );
         when( dimensionIdentifierConverter.fromString( programs, dimension ) ).thenReturn( deDimensionIdentifier );
-        when( (dataQueryService.getDimension( (String) deDimensionIdentifier.getDimension(),
+        when( (dataQueryService.getDimension( deDimensionIdentifier.getDimension().getUid(),
```

Comment:
I found here some issues in old code: 
lines: 171, 237, 313, 317, 361, 385, 417, 466, 493, most of them unused variables, can you please fix it?